src/Components/WeatherDetails/WeatherAdditionalDetails.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+
+import WeatherAdditionalDetailsItem from './WeatherAdditionalDetailsItem';
+import { FadeInTop } from '../../common/Animations/Animations';
+
+class WeatherAdditionalDetails extends Component {
+  state = {
+    active: false,
+    data: null,
+  };
+
+  handleStateReachedData = data => {
+    this.setState({ data });
+  };
```

Comment:
this method has wrong name which is hard to understand - this should warn you, that this method may be unnecessary. check next comment